mmrazor/models/task_modules/estimators/counters/flops_params_counter.py

```diff
@@ -0,0 +1,467 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import sys
+from functools import partial
+
+import torch
+import torch.nn as nn
+
+from mmrazor.registry import TASK_UTILS
+
+
+def get_model_complexity_info(model,
+                              input_shape,
+                              spec_modules=[],
```

Comment:
spec_modules -> custom_keys to support prefix, ref to mmcv::mmcv/runner/optimizer/default_constuctor.py

Reply:
Now support counting flops with a specified scope, e.g. spec_modules = ['backbone']